apps/storybook/src/DashboardPageBody.stories.tsx

```diff
@@ -17,7 +17,11 @@ const props = (): ComponentProps<typeof DashboardPageBody> => ({
       id: 'uuid-1',
       created: new Date().toISOString(),
       type: 'News' as const,
-      title: "Coordinating different research interests into Parkinson's",
+      title: 'Learn about Protocols.io, an ASAP preferred tool',
+      shortText:
+        'Discover current and planned tools (e.g., animal & cell models, antibodies, vectors, tissues, etc.) in a sortable table. This will be a living reference.',
+      link: text('Link', 'https://picsum.photos/200'),
```

Comment:
Where possible we try and use http://example.com as its exactly their use case (https://en.wikipedia.org/wiki/Example.com)

Reply:
Internal links we generally use #wrong